R/get_about_pages.r

```diff
@@ -161,24 +168,11 @@ extract_about_links <- function(base_url, timeout_thres = 10) {
           filter(!is.na(href)) %>%
           distinct() 
 
-        # To make link formatting not off when page uses an absolute reference        
-        if (str_detect(about_links$href, "http")) {
-
-        # TRUE (absolute path)
-        about_links <- about_links %>%
-          mutate(link = href) %>%
-          mutate(href = "Absolute link") %>%
-          select(href, link)
-        
-        } else {
-        
-        # FALSE (relative path)
+        # To make link formatting not off when page uses an absolute reference   
         about_links <- about_links %>%
-          mutate(href = str_replace_all(href, "/", "")) %>%
-          mutate(link = glue("{base_url}{href}")) %>%
+          mutate(link = url_absolute(href, raw_url)) %>%
```

Comment:
This is kind of duplicate with what we do later in line 223. we parse the link relative to the base in both cases. not sure that's efficient.

Reply:
Since base_url is used across `extract_about_links()` and `find_about_link()` we need to solve this problem in both cases. Inspired your PR, I worked on it, and my new PR fixes the problem on both occasions.